### DISCUSSION TITLE
Add some attributs for values specifics

### DISCUSSION THREAD

**Discussion by ZolOnTheNet** [Mezryss/FVTT-Genesys#109] [Ideas]
Similarly to the DnD5 version on Foundry, would it be possible to have two or three resources to put, preferably on the first page, of the character sheet? This could represent strength points, dream points, and magic points. As in DnD, the ability to rename the resource would allow giving meaning to it in the vtt-token-action-hud-genesys.

It would be really nice to be able to do RPG hacks with this system.

Thank for yours jobs,

![Proposition Ressource](https://github.com/Mezryss/FVTT-Genesys/assets/103053124/57e69dd3-2c17-4c63-8afb-72ecdacfe65e)

**Comment by Cristalnoir** [comment]
excellent idea, I myself will need it. No corruption, no madness, etc. This will allow you to personalize
